fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -152,25 +154,24 @@ public class FDBDirectory extends Directory  {
 
     private final Map<BitSet, byte[]> fieldInfosDataMap;
 
+    private final boolean primaryKeySegmentIndexEnabled;
+    @Nullable
+    private LucenePrimaryKeySegmentIndex primaryKeySegmentIndex;
+
     public FDBDirectory(@Nonnull Subspace subspace, @Nonnull FDBRecordContext context) {
```

Comment:
It looks like this only has a few test-usages, perhaps it would make sense to add `primaryKeySegmentIndexEnabled` as an argument to this constructor

Reply:
Added. I didn't change any of those callers to pass anything other than `false` though.